core/src/avm1.rs

```diff
@@ -157,11 +178,11 @@ impl Avm1 {
         Ok(())
     }
 
-    pub fn resolve_slash_path<'gc>(
-        start: DisplayNode<'gc>,
-        root: DisplayNode<'gc>,
+    pub fn resolve_slash_path<'dgc>(
```

Comment:
I don't think there's a guarantee that the lifetime received here is the same as the `'gc` on the entire avm root. This code didn't compile when I tried it last night.

We could just split it off into a separate `impl` block with an anonymous root lifetime, if you want to avoid the name clashing.

Reply:
Huh! It was in the middle of all the other changes so I guess it was just a symptom of those not being finished, perhaps. Fixing <3